### DISCUSSION TITLE
Image Registration for Satellite data

### DISCUSSION THREAD

**Discussion by kanishk-varshney** [kornia/kornia#1668] [Questions]
Hey, 
New to Kornia! I wanted to try image registration for satellite imagery.
Typical remote sensing input format is geotiff, and expecting same output format. Kornia doesn't support this use case, any leads on how to start and implement this?

**Comment by edgarriba** [comment]
@varskann I'm currently working on a Rust backend to load images with kornia -- https://github.com/kornia/kornia-rs

we could explore from there to load geotiff images. Worth you give it a shot
